packages/visualizations/addon/components/navi-vis-c3-chart.js

```diff
@@ -44,9 +44,12 @@ export default C3Chart.extend({
   dataDidChange() {
     // Add custom classes to each data series for easier reference and coloring
     let { data, dataClasses } = getProperties(this, 'data', 'dataClasses'),
-        dataWithClasses = Ember.assign({}, { classes: dataClasses }, data);
+        dataWithClasses = Ember.assign({}, { classes: dataClasses }, data),
+        dataWithClasses2 = Ember.assign({}, dataWithClasses);
```

Comment:
can we make this `dataWithClasses2` variable more descriptive

Reply:
why do we need dataWithClasses2?